transformation/tests/org.jboss.tools.fuse.transformation.editor.tests.integration/src/main/java/org/jboss/tools/fuse/transformation/editor/FuseProject.java

```diff
@@ -0,0 +1,56 @@
+/******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. and others.
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors: JBoss by Red Hat - Initial implementation.
+ *****************************************************************************/
+package org.jboss.tools.fuse.transformation.editor;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.junit.rules.ExternalResource;
+
+public class FuseProject extends ExternalResource {
```

Comment:
I thought about that, but I was trying to avoid what seemed like an odd dependency to another test project that had nothing to do with transformations.  Plus, when I first created the test, I was planning on using multiple instances of FuseProject, each with different POM contents, and the auto-setting of those contents in your class's constructor shied me away from that.  I guess there's no great reason not to use it though, and would avoid a mostly duplicate class.  No one has problems with the weird dependency?  If not, I'll change this.